lib/puppet/parser/functions.rb

```diff
@@ -272,7 +272,7 @@ def get_function(name, environment)
 
   class Error
     def self.is4x(name)
-      raise Puppet::ParseError, "#{name}() can only be called using the 4.x function API. See Scope#call_function"
+      raise Puppet::ParseError, _("%{name}() can only be called using the 4.x function API. See Scope#call_function") % { name: name }
```

Comment:
Can we just tell the translators to not translate anything that includes an `_`?